src/base_circuitpython/displayio/group.py

```diff
@@ -69,9 +69,8 @@ def x(self):
 
     @x.setter
     def x(self, val):
-        changed = val != self.__x
-        self.__x = val
-        if changed:
+        if val != self.__x:

```

Comment:
_&ast;nit&ast;_
I would change the order here to:
`if self.__x != val:`
a tiny bit easier to read when the property is on the left and the new value is on the right.

Reply:
agreed